### ISSUE TITLE
ansible-lint is checking for 601 on commented lines while running play

### ISSUE THREAD

**Issue by bgajjela** [ansible/ansible-lint#680]
# Issue Type
- Bug report
- Feature request
bug 
# Ansible and Ansible Lint details

```
ansible --version
ansible-lint --version
```
ansible-lint==4.1.0

- ansible installation method: one of source, pip, OS package
pip
- ansible-lint installation method: one of source, pip, OS package
pip

# Desired Behaviour

Please give some details of the feature being requested
or what should happen if providing a bug report

Possible security bugs should be reported via email to `security@ansible.com`

# Actual Behaviour (Bug report only)
ansible-lint is checking for 601 on commented lines while running play with --check option. 

Please give some details of what is actually happening.
Include a [minimum complete verifiable example](http://stackoverflow.com/help/mcve)
with:
- playbook
- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`
